__test__/pet.test.js

```diff
@@ -0,0 +1,163 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+
+  it('returns an object', () => {
+    expect(new Pet('Goose')).toBeInstanceOf(Object);
+  });
+
+  it('sets the name property', () => {
+    const pet = new Pet('Goose');
+    expect(pet.name).toEqual('Goose');
+  });
+
+  it('has a initial age of 0', () => {
+    const pet = new Pet('Goose');
+    expect(pet.age).toEqual(0);
+  });
+});
+
+describe('growUp', () => {
+
+  it('throws an error if the pet is not alive', () => {
+    const pet = new Pet('Goose');
+    pet.age = 30;
+    expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+  }); 
+
+  it('increase the age by 1', () => {
+    const pet = new Pet('Goose');
+    pet.growUp();
+    expect(pet.age).toEqual(1);
+  });
+
+  it('increase the hunger value by 5', () => {
+    const pet = new Pet('Goose');
+    pet.growUp();
+    expect(pet.hunger).toEqual(5);
+  });
+
+  it('decreases the fitness value by 3', () => {
+    const pet = new Pet('Goose');
+    pet.growUp();
+    expect(pet.fitness).toEqual(7);
+  });
+});
+
+describe('walk', () => {
+
+  it('throws an error if the pet is not alive', () => {
+    const pet = new Pet('Goose');
+    pet.age = 30;
+    expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+  }); 
+
+  it('increase fitness by 4', () => {
+    const pet = new Pet('Goose');
+    pet.fitness = 4;
+    pet.walk();
+    expect(pet.fitness).toEqual(8);
+  });
+  
+  it('increases fitness by a maximum to 10', () => {
+    const pet = new Pet('Goose');
+    pet.fitness = 8;
+    pet.walk();
+    expect(pet.fitness).toEqual(10);
+  });
+});
+
+describe('feed', () => {
+ 
+  it('throws an error if the pet is not alive', () => {
+    const pet = new Pet('Goose');
+    pet.age = 30;
+    expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+  }); 
+
+  it('decreases hunger by 3', () => {
+    const pet = new Pet('Goose');
+    pet.hunger = 5;
+    pet.feed();
+    expect(pet.hunger).toEqual(2);
+  });
+
+  it('decrease hunger by a minimum to 0', () => {
+    const pet = new Pet('Goose');
+    pet.hunger = 2;
+    pet.feed();
+    expect(pet.hunger).toEqual(0);
+  });
+});
+
+describe('checkUp', () => {
+
+  it('throws an error if the pet is not alive', () => {
+    const pet = new Pet('Goose');
+    pet.age = 30;
+    expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+  }); 
+
+  it('if fitness is 3 or less', () => {
+    const pet = new Pet('Goose');
+    pet.fitness = 3;
+    pet.checkUp();
+    expect(console.log("I need a walk"))
+  });
+
+  it('if hunger is 5 or more', () => {
+    const pet = new Pet('Goose');
+    pet.hunger = 5;
+    pet.checkUp();
+    expect(console.log("I am hungry"))
+  });
+
+  it('if fitness is 3 or less AND hunger is 5 or more', () => {
+    const pet = new Pet('Goose');
+    pet.fitness < 3 && pet.hunger > 5;
```

Comment:
this line doesn't do anything, it will check if pet.fitness is less than 3 and pet.hunger is more than 5, which will evaluate to either true or false, either way the code underneath it will run as it's not in an if statement. In this test, you should be setting `pet.fitness = 2` (or any value less than 3 and more than 0) and setting `pet.hunger = 6` (or any value larger than 5 and less than 10)